src/components/transport_manager/include/transport_manager/transport_manager_impl.h

```diff
@@ -350,7 +350,8 @@ class TransportManagerImpl
   DeviceInfoList device_list_;
 
   void AddConnection(const ConnectionInternal& c);
-  void RemoveConnection(uint32_t id);
+  void RemoveConnection(uint32_t id,
```

Comment:
It was non const so I decided to leave that way. However if there are some other changes to be made I will change this to const together with all the eligible methods in this header.